### DISCUSSION TITLE
ClassName Attribute

### DISCUSSION THREAD

**Discussion by ghost** [uxsoft/Fable.AntDesign#39] [Q&A]
How do I add a class name to the AntD components?

**Comment by uxsoft** [comment]
Hi,

If you're using SwiftUI style bindings (Fable.AntD), there is an explicit attribute where Ant Design has it described in the documentation (Divider, Drawer, Layout...)

```fsharp
AntDivider()
    .className("my-class")
    .[[]]
```

If this attribute isn't present, you can still set the prop, in most cases, it should propagate.

```fsharp
AntButton()
    .set("className", "my-class")
    .[[]]
```

In computation expression based bindings (Fable.AntDesign) the attribute is common to all controls

```fsharp
button {
    className "my-class"
}
```
